osu.Game.Rulesets.Catch.Tests/Mods/TestSceneCatchModPerfect.cs

```diff
@@ -50,5 +50,7 @@ public void TestJuiceStream(bool shouldMiss)
         // We only care about testing misses, hits are tested via JuiceStream
         [TestCase(true)]
         public void TestTinyDroplet(bool shouldMiss) => CreateHitObjectTest(new HitObjectTestData(new TinyDroplet { StartTime = 1000 }), shouldMiss);
+
+        protected override Ruleset CreateRuleset() => new CatchRuleset();
```

Comment:
Not sure about the placement of these, I'd probably want them to be closer to the top (before ctor?)